server/data/data-api/src/main/java/org/apache/james/user/api/UsersRepository.java

```diff
@@ -142,4 +142,10 @@
      */
     boolean isAdministrator(String username) throws UsersRepositoryException;
 
+    /**
+     * @return true if one can use {@link UsersRepository#updateUser(User)} {@link UsersRepository#addUser(String, String)}
```

Comment:
Why?
 - It would be a mess for an injection point of view (How UserProvisionningFilter would behave if it do not find the interface for creating users?)
 - I'm not shocked by the way an implementation is partial because it is read-only...